### ISSUE TITLE
Ivory coast links to north korea

### ISSUE THREAD

**Issue by M350Z01K** [caldwellst/whoville#8]
Congrats for the package! Very useful! 

However, the matching could be improved:

```whoville::names_to_iso3('ivory coast', language = 'en')```

```>'ivory coast' has no exact match. Closest name found was 'north korea'.```
```>[1] "PRK"```

Playing around with the p parameter does not help

**Comment by ElliottMess** [comment]
Thanks for the feedback and glad that you appreciate the package (: 

Fussy matches, like we use here, are notoriously difficult to calibrate to return useful results. The goal of the error message was more to find potential alternative while informing the user of the lack of exact match.

**Comment by M350Z01K** [comment]
Yeah I know the limitations of Levenshtein and co. Maybe an approach with "blocking" such as in the RecordLinkage package: https://cran.r-project.org/web/packages/RecordLinkage/index.html would do the trick. Or maybe it's a bit overkill. You decide.

**Comment by M350Z01K** [comment]
By the way, are you still working for WHO?

I am a consultant in data science and I feel quite alone here.

**Comment by mayeulk** [comment]
Adding "Ivory Coast" in "alt_countries.xlsx" (Row "Cote d'Ivoire") shoul do the trick, no?
